ui/views_test.py

```diff
@@ -72,6 +72,21 @@ def user_admin_list_data():
     return SimpleNamespace(video=video, moira_list=moira_list, collection=collection)
 
 
+@pytest.fixture
+def post_data(logged_in_apiclient):
+    """Fixture for testing collection creation using valid post data"""
+    client, user = logged_in_apiclient
+    url = reverse('models-api:collection-list')
+
+    input_data = {
+        'owner': user.id,
+        'title': 'foo title',
+        'view_lists': [],
+        'admin_lists': []
+    }
+    return SimpleNamespace(client=client, user=user, url=url, data=input_data)
```

Comment:
How about a compromise with no test class, but just the input_data as a fixture (still called `post_data` because eventually we might need `put_data` and `patch_data`), and url specified in each test for clarity:
```
@pytest.fixture
def post_data(logged_in_apiclient):
    """Fixture for testing collection creation using valid post data"""
    _, user = logged_in_apiclient

    input_data = {
        'owner': user.id,
        'title': 'foo title',
        'view_lists': [],
        'admin_lists': []
    }
    return input_data

def test_collection_viewset_create_as_normal_user(post_data, logged_in_apiclient):
    client, _ = logged_in_apiclient
    url = reverse('models-api:collection-list')
    result = client.post(url, post_data, format='json')
    assert result.status_code == status.HTTP_403_FORBIDDEN
```